### PR TITLE
docs: Add changelog 1.10.0

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

**Review by boojack — APPROVED** [review]

**Review by ecmadao — APPROVED** [review]

**Review by ecmadao — APPROVED** [review]

File: content/changelog/bytebase-1-10-0.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+author: Ningjing
+published_at: 2022/12/29 17:00:00
+description: "- Workspace owner and DBA can now view audit logs. - Support CRUD database role (only for PostgreSQL) via Terraform Provider. - Support viewing and editing primary keys and foreign keys in the Schema editor for MySQL. - Support MySQL/TiDB SSL connection."
+---
+
+## 🚀 New Features
+
+- Workspace owner and DBA can now view audit logs.
+- Support CRUD database role (only for PostgreSQL) via Terraform Provider. Read the [docs](https://registry.terraform.io/providers/bytebase/bytebase/latest/docs/resources/database_role).
+- Support viewing and editing primary keys and foreign keys in the Schema editor for MySQL.
```
Comment (tianzhou) [inline content/changelog/bytebase-1-10-0.md:12]:
```suggestion
- Support viewing and editing primary keys and foreign keys in the Schema Editor for MySQL.
```

File: content/changelog/bytebase-1-10-0.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Support MySQL/TiDB SSL connection.
+- Support DML with large SQL (up to 100M).
```
Comment (tianzhou) [inline content/changelog/bytebase-1-10-0.md:14]:
```suggestion
- Support DML with large SQL (up to 100MB).
```

File: content/changelog/bytebase-1-10-0.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- Speed up the issue detail page.
+
+## 🐞 Notable bug fixes
+
+- Fixed: querying Snowflake from SQL Editor.
+
+## 🎠 Community
+
+- We have revamped our [GitHub README page](https://github.com/bytebase). Please do check it out!
+- Thanks to [@piglig](https://github.com/piglig) for the PRs：
```
Comment (tianzhou) [inline content/changelog/bytebase-1-10-0.md:32]: Needs indentation

CleanShot 2022-12-29 at 17 58 04

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**